integration/csharp/tests/Duckstax.Otterbrix.Tests/WrapperTest.cs

```diff
@@ -0,0 +1,286 @@
+namespace Duckstax.Otterbrix.Tests;
+
+using Duckstax.EntityFramework.Otterbrix;
+
+public class Tests
+{
+    private string GenerateID(int num) {
+        string res = num.ToString();
+        while (res.Length < 24) {
+            res = "0" + res;
+        }
+        return res;
+    }
+
+    [Test]
+    public void Base() {
+        OtterbrixWrapper otterbrix = new OtterbrixWrapper(Config.DefaultConfig(), "TestDatabase", "TestCollection");
+        {
+            string query = "INSERT INTO TestDatabase.TestCollection (_id, name, count) VALUES ";
+            for (int num = 0; num < 100; ++num) {
+                query += ("('" + GenerateID(num + 1) + "', " + "'Name " + num + "', " + num + ")" +
+                          (num == 99 ? ";" : ", "));
+            }
+            CursorWrapper cursor = otterbrix.Execute(query);
+            Assert.IsTrue(cursor != null);
+            Assert.IsTrue(cursor.IsSuccess());
+            Assert.IsTrue(cursor.Size() == 100);
+        }
+        {
+            string query = "SELECT * FROM TestDatabase.TestCollection;";
+            CursorWrapper cursor = otterbrix.Execute(query);
+            Assert.IsTrue(cursor != null);
+            Assert.IsTrue(cursor.IsSuccess());
+            Assert.IsFalse(cursor.IsError());
```

Comment:
I think it cannot error if previous assertion passes - it's not a Shroedinger `s cat. Same for other assertions. It can save some CI time.

Reply:
The way i build it, it can, there is a case, when IsSuccess() will be false, but IsError() will also be false.
If we select something but nothing was selected, operation won't be successful, because nothing was selected, but also there won't be any errors, because everything worked fine.
It done that way to simplify some checks when there is need to see if something was selected or not